src/NLog/Targets/DatabaseParameterInfo.cs

```diff
@@ -98,6 +99,13 @@ public DatabaseParameterInfo(string parameterName, Layout parameterLayout)
         /// <docgen category='Parameter Options' order='10' />
         [DefaultValue(0)]
         public byte Scale { get; set; }
+
+        /// <summary>
+        /// Gets or sets the database parameter type.
+        /// </summary>
+        /// <docgen category='Parameter Options' order='10' />
+        [DefaultValue(DbType.AnsiString)]
```

Comment:
I think this is inconsistent with

The default is NVarChar. (https://msdn.microsoft.com/en-us/library/system.data.sqlclient.sqlparameter.dbtype(v=vs.110).aspx)

Which is `String` IMO (see https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/sql-server-data-type-mappings)

Proposal: make nullable and only set if non-null in databasetarget